### ISSUE TITLE
二阶段训练代码有错误

### ISSUE THREAD

**Issue by zhanghang-official** [OpenGVLab/Ask-Anything#108]
二阶段训练需要
from models.videochat2_pt import VideoChat2_pt
但是发布的版本是
from models.videochat2_it import VideoChat2_it
麻烦修改下。（是不是你们设置的小彩蛋）

**Comment by Andy1621** [comment]
Get it! Thanks for your reply!

**Comment by robotNo201** [comment]
> 二阶段训练需要 from models.videochat2_pt import VideoChat2_pt 但是发布的版本是 from models.videochat2_it import VideoChat2_it 麻烦修改下。（是不是你们设置的小彩蛋）

yea, definitely, guys from lab will make the code as ambiguous as they can, cuz they get the money, and they dont want to share the code with you, i understand.

**Comment by Andy1621** [comment]
> > 二阶段训练需要 from models.videochat2_pt import VideoChat2_pt 但是发布的版本是 from models.videochat2_it import VideoChat2_it 麻烦修改下。（是不是你们设置的小彩蛋）
> 
> yea, definitely, guys from lab will make the code as ambiguous as they can, cuz they get the money, and they dont want to share the code with you, i understand.

I don't think so. I have tried my best to make all my jobs open-sourced and reproducible. I recognize that there may be some bugs when cleaning the code, but welcome to open an issue and I will give the answer.
